lib/segment/src/index/field_index/full_text_index/compressed_posting/mod.rs

```diff
@@ -0,0 +1,40 @@
+mod compressed_chunks_reader;
+pub mod compressed_common;
+mod compressed_posting_iterator;
+pub mod compressed_posting_list;
+pub mod compressed_posting_visitor;
+
+//
+//                               Posting Iterator
```

Comment:
I suggest to use real struct names for easier navigating.
I.e. `Posting Iterator` → `CompressedPostingIterator`, `Chunk Visitor` → `CompressedPostingVisitor`, and so on.